db/migrate/20210204120501_add_status_to_content_pages.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToContentPages < ActiveRecord::Migration[6.1]
+  change_table :content_pages, bulk: true do |t|
+    t.string :published_status
```

Comment:
what about boolean for published status,  boolean  for any other statuses, inactive/active, i believe this is how wordpress works, once its published its published but it can be inactive

Reply:
I thought about numbers or booleans, but an enum with symbols would be good, because then they could be converted to strings in the UI.

Kinda regret adding this migration too early !

Reply:
Lets do this when the story is given to us

Reply:
Given this migration is not used in this PR, it should probably be removed for now and wait for the story to create it.